### PR TITLE
185.follower following lists

### LINKED ISSUE #185: Follower and Following Lists
[//]: # (These are comments and are used for reference and will not show up in the PR)

User Requirement: N/A

[//]: # (Please Add labels for type of issue)

Description: Cool and spicy extra feature to display your list of followers and who you follow

[//]: # (If an issue is a bug or defect, please list steps to be able to reproduce bug and a screenshot of the defect i.e terminal message)

Story Points: 3

Related Issues: N/A

[//]: # (Notes:)
[//]: # (- Testing and Documentation tasks should always be assumed, but only need to be explicitly stated if there is something that ~~does not~~ need modification. For example,)
[//]: # (some Issues/Stories may not require one of Unit or Intent tests, and this should be indicated in the task list.)
[//]: # (- Code Reviews are also always assumed to be required, but an issue should only indicate in the description if a review is not required, but someone still shouldn't)
[//]: # (merge their own pull request)

### PR DESCRIPTION
**Pull request by gatilogo**
[//]: # (These are comments and are used for reference and will not show up in the PR)

Description: Completely revamped profile pages

[//]: # (Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change)

[//]: # (automatically closes given issue number)
Closes #185 

Proof of Concept

[//]: # (Attach Image of feature added or fix applied and working)
[//]: # (Please provide images of tests passing)
image
image

Checklist

[//]: # (Add an x between square brackets to mark item as done)

- [x] Follows style guidelines and is **properly commented**
- [x] Code changes generate no new warnings
- [ ] Testing (Unit and/or Intent)
- [x] Documentation (UML, Wiki, etc.)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -21,7 +21,7 @@ android {
         sourceCompatibility = 1.8
         targetCompatibility = 1.8
     }
-    testOptions{
+    testOptions {
```
Comment (jacobwong98) [inline app/build.gradle]: nice

File: app/src/main/java/com/example/vybe/ConnectionsPageAdapter.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    public ConnectionsPageAdapter(@NonNull FragmentManager fm, int behavior, Context context, User user) {
+        super(fm, behavior);
+        this.resources = context.getResources();
+        this.user = user;
+    }
+
+    @Override
+    public int getCount() {
+        return 2;
+    }
```
Comment (jacobwong98) [inline app/src/main/java/com/example/vybe/ConnectionsPageAdapter.java]: what?

Comment (TheBroMoe) [inline app/src/main/java/com/example/vybe/ConnectionsPageAdapter.java]: Does this need to be a method? or can it be changed to a variable

File: app/src/main/java/com/example/vybe/MyVibesActivity.java [hunk 3]
```diff
@@ -104,7 +104,14 @@ protected void onCreate(Bundle savedInstanceState) {
                         .addOnSuccessListener((DocumentSnapshot doc) -> {
                             String username = (String) doc.get("username");
                             String email = (String) doc.get("email");
-                            intent.putExtra("user", new User(username, email));
+                            ArrayList<String> followers = (ArrayList) doc.get("followers");
+                            ArrayList<String> following = (ArrayList) doc.get("following");
+                            User user = new User(username, email);
+                            user.setFollowers(followers);
+                            user.setFollowing(following);
```
Comment (jacobwong98) [inline app/src/main/java/com/example/vybe/MyVibesActivity.java]: please remove

**Pull request merged**